appveyor-build.cmd

```diff
@@ -20,7 +20,7 @@ if not exist %_msbuildexe% echo Error: Could not find MSBuild.exe. && goto :eof
 set _ngenexe="%SystemRoot%\Microsoft.NET\Framework\v4.0.30319\ngen.exe"
 if not exist %_ngenexe% echo Error: Could not find ngen.exe. && goto :failure
 
-.\.nuget\NuGet.exe restore packages.config -PackagesDirectory packages
+.\.nuget\NuGet.exe restore packages.config -PackagesDirectory packages -ConfigFile .nuget\NuGet.Config
```

Comment:
Nuget package restore is hooked in automatically in FSharpSource.targets, see the "BeforeBuild" target (toward the bottom).  That invocation needs to be update with `-ConfigFile`, too.